client/config.go

```diff
@@ -0,0 +1,111 @@
+package client
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	"istio.io/manager/apiserver"
+	"istio.io/manager/model"
+)
+
+func (m *ManagerClient) GetConfig(key model.Key) (*apiserver.Config, error) {
+
```

Comment:
super nit: I think the most common style is no leading whitespace at the start of functions?